src/main/java/org/elasticsearch/index/query/GeoPolygonFilterParser.java

```diff
@@ -76,8 +76,8 @@ public Filter parse(QueryParseContext parseContext) throws IOException, QueryPar
 
         List<GeoPoint> shell = Lists.newArrayList();
 
-        boolean normalizeLon = true;
-        boolean normalizeLat = true;
+        boolean normalizeLon = false;
```

Comment:
I agree. ```normalize``` should be documented.  The reason behind changing defaults to ```false``` is this commit changes ```pointInPolygon()``` logic to translate the vertex coordinates of the search poly and the candidate point to a great-circle/sphere coordinate system.  ```normalize()``` translates to standard lat/lon coordinate system.  So we have a situation for this filter where a call to ```normalize()``` induces unnecessary computation since everything will be translated back to great-circle anyway.